src/sagemaker/jumpstart/curated_hub/parsers.py

```diff
@@ -0,0 +1,422 @@
+        if document["GatedBucket"]
+        else f"jumpstart-cache-prod-{region}"
+    )
+    document["HostingArtifactUri"] = s3_path_join(
+        "s3://", content_bucket, model_specs.hosting_artifact_key
+    )
+    document["HostingArtifactS3DataType"] = studio_specs.get("inferenceArtifactS3DataType")
+    document["HostingArtifactCompressionType"] = studio_specs.get(
+        "inferenceArtifactCompressionType"
+    )
+
+    document["HostingScriptUri"] = s3_path_join(
+        "s3://", content_bucket, model_specs.hosting_script_key
+    )
+    document["InferenceDependencies"] = model_specs.inference_dependencies
+    document["InferenceEnvironmentVariables"] = model_specs.inference_environment_variables
+    document["TrainingSupported"] = model_specs.training_supported
+    document["IncrementalTrainingSupported"] = model_specs.incremental_training_supported
+    document[
+        "DynamicContainerDeploymentSupported"
+    ] = model_specs.dynamic_container_deployment_supported
+    document["HostingPrepackedArtifactUri"] = (
+        s3_path_join("s3://", content_bucket, model_specs.hosting_prepacked_artifact_key)
+        if model_specs.hosting_prepacked_artifact_key is not None
+        else None
+    )
+    document["HostingPrepackedArtifactVersion"] = model_specs.hosting_prepacked_artifact_version
+    document["HostingUseScriptUri"] = model_specs.hosting_use_script_uri
+    document["HostingEulaUri"] = (
+        s3_path_join("s3://", content_bucket, model_specs.hosting_eula_key)
+        if model_specs.hosting_eula_key is not None
+        else None
+    )
+    if model_specs.hosting_model_package_arns and region in model_specs.hosting_model_package_arns:
+        document["HostingModelPackageArn"] = model_specs.hosting_model_package_arns[region]
+    document["DefaultInferenceInstanceType"] = model_specs.default_inference_instance_type
+    document["SupportedInferenceInstanceTypes"] = model_specs.supported_inference_instance_types
+    document["SageMakerSdkPredictorSpecifications"] = model_specs.predictor_specs
+    document["InferenceVolumeSize"] = model_specs.inference_volume_size
+    document["InferenceEnableNetworkIsolation"] = model_specs.inference_enable_network_isolation
+    document["ResourceNameBase"] = model_specs.resource_name_base
+    document["DefaultPayloads"] = {
+        alias: walk_and_apply_json(payload.to_json(), snake_to_upper_camel)
+        for alias, payload in model_specs.default_payloads.items()
+    }
+    document["HostingResourceRequirements"] = model_specs.hosting_resource_requirements
+    document[
+        "HostingInstanceTypeVariants"
+    ] = model_specs.hosting_instance_type_variants.regionalize(region)
+    default_training_dataset_key = studio_specs.get("defaultDataKey")
+    document["DefaultTrainingDatasetUri"] = (
+        s3_path_join("s3://", content_bucket, model_specs.hosting_prepacked_artifact_key)
+        if default_training_dataset_key
+        else None
```

Comment:
Same questions for these parsers: Can these values be `None` in the `import` request?

Reply:
Keys with None values will not be serialized!